bedrock/mozorg/templates/mozorg/home/home-fr.html

```diff
@@ -18,6 +18,10 @@
   {% if switch('firefox-daylight-launch-banner') %}
     {{ css_bundle('daylight-launch-banner') }}
   {% endif %}
+
+  {% if switch('firefox-daylight-promo-banner') %}
```

Comment:
Should this be a CSS bundle instead?

![image](https://user-images.githubusercontent.com/400117/94690263-ccd43800-0327-11eb-84a6-a647d908d265.png)

Reply:
Oops, botched copypasta.